### ISSUE TITLE
Tab focus state risks failing WCAG 2.2 criterion 14.2.13: Focus appearance

### ISSUE THREAD

**Issue by owenatgov** [alphagov/govuk-frontend#4471]
## Description of the issue
The [tabs component](https://design-system.service.gov.uk/components/tabs/) doesn't meet the minimum requirements for [WCAG 2.2 criterion 14.2.13: focus appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html) under circumstances where tab text is very short. This is because:

1. the component uses link decoration with black text, reducing the contrast change between focused and unfocused to meet the criterion
2. for "active" tabs (white background) there is no text decoration, reducing the possible area focus area required to meet the criterion
3. for "inactive" tabs (grey background) the contrast between the grey background and the focus yellow is too low

## Steps to reproduce the issue
See [the testing spreadsheet, including the focus appearance pass fail calculator (internal only)](https://docs.google.com/spreadsheets/d/1PpKvhCGZJb1j0ZUUY2ZuA0CBQeu04yFD1bumiL8wGX8/edit#gid=732550912) for more details on the circumstances where this issue occurs

## Further reading
- https://github.com/alphagov/govuk-design-system/issues/3007 - related issue on focus state rationalisation
- https://github.com/alphagov/govuk-frontend/issues/4470 - may inadvertently solve this issue
